fulp_modules/features/mentors/mentor_state.dm

```diff
@@ -0,0 +1,12 @@
+/**
+ * tgui state: admin_state
+ *
+ * Checks that the user is an admin, end-of-story.
```

Comment:
shouldn't this be mentor not admin here.

Reply:
error in copypasta